Source/Microsoft.Teams.Apps.FAQPlusPlus/Cards/CardHelper.cs

```diff
@@ -87,15 +87,25 @@ public static string GetTicketDisplayStatusForSme(TicketEntity ticket)
         }
 
         /// <summary>
-        /// Returns a string that will display the given date and time in the user's local time zone, when placed in an adaptive card.
+        /// Returns a string that will display the given date and time in the user's local time zone in a thumbnail card.
         /// </summary>
         /// <param name="dateTime">The date and time to format.</param>
         /// <param name="userLocalTime">The sender's local time, as determined by the local timestamp of the activity.</param>
         /// <returns>A datetime string.</returns>
         public static string GetFormattedDateInUserTimeZone(DateTime dateTime, DateTimeOffset? userLocalTime)
         {
-            // Adaptive card on mobile has a bug where it does not support DATE and TIME, so for now we convert the date and time manually.
-            return dateTime.Add(userLocalTime?.Offset ?? TimeSpan.FromMinutes(0)).ToShortDateString();
+            return dateTime.Add(userLocalTime?.Offset ?? TimeSpan.FromMinutes(0)).ToString("dddd, dd MMMM yyyy", CultureInfo.InvariantCulture);
```

Comment:
Pls use a built-in format. This custom format is not correct for all locales. For example en-us formats dates as "month day, year", not "day month year"

Reply:
We have updated the date format according to how Adaptive card is showing it. We have observed, for different cultures it is not changing the format of date and to make the consistency over the app we have used the same format here. 

Let us know if this looks good.
Thanks.

Reply:
@hunaidhanfee20 what does this mean? "We have observed, for different cultures it is not changing the format of date"

Below we the date is formatted as `DATE(" + utcString + ", SHORT)`. Are you claiming that when the client language is changed to say French, the format doesn't change?

Also, `ToString("dddd, dd MMMM yyyy", CultureInfo.InvariantCulture)` produces a "long" date, and in an invariant culture, which doesn't seem correct at all. The change in this PR turns a culture-sensitive short date to an invariant longer date.

Reply:
Below we the date is formatted as DATE(" + utcString + ", SHORT). Are you claiming that when the client language is changed to say French, the format doesn't change? => **Yes**
PFB screenshot for French language.
![French_adaptive_card](https://user-images.githubusercontent.com/60421876/121418776-e9317580-c988-11eb-901f-35e636b16f44.png)

Also, ToString("dddd, dd MMMM yyyy", CultureInfo.InvariantCulture) produces a "long" date, and in an invariant culture, which doesn't seem correct at all. The change in this PR turns a culture-sensitive short date to an invariant longer date. => **Reverted to shortdate, except for Arabic as it was showing date as Islamic calendar.**